crates/bevy_pbr/src/light.rs

```diff
@@ -57,7 +112,7 @@ impl Default for PointLight {
     fn default() -> Self {
         PointLight {
             color: Color::rgb(1.0, 1.0, 1.0),
-            intensity: 800.0, // Roughly a 60W non-halogen incandescent bulb
+            intensity: 4000.0, // Roughly a 300W non-halogen incandescent bulb
```

Comment:
could we use a led light instead of an incandescent bulb in the comment? 
4000 lumens seems a bit high, not your off the shelf lightbulb